eng/tools/RunTests/ProcessResult.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Due to folder move and file-scoped namespace change, it's much easier to look at individual commits to see what changed here.